contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -312,6 +314,17 @@ def _show_capture_window(self):
         logging.info('Showing capture window')
         self.find_control("TabItem", "Capture").click_input()
 
+    def _show_capture_options_dialog(self):
```

Comment:
This and the method below contains code copied from `_set_capture_options`. If you choose to keep these methods (see my comment to the PR), please remove the duplicated code from `_set_capture_options` and call these two methods instead.

Reply:
I didn't see it! I copied from SetAndCheckMemorySamplingPeriod. I should have done in a different previous PR but it's late now. I will erase them in a following PR together with the refactor needed for the memory part. Since the tests have passed, I'd prefer to do it in a following smaller PR.